### ISSUE TITLE
project previews vs project detail pages

### ISSUE THREAD

**Issue by krispy1298** [alexsomeoddpilot/sop-gulp-static#3]
do you think each "block" should be its own file within a client folder?

work
—CAT
——block 1 (used in homepage)
——block 2
——block 3

or should each client be a file and the file has a loop of stuff and the first loop is pushed to homepage?

1st scenario is easier to control, but im guessing harder to configure gulp?

**Comment by alexsomeoddpilot** [comment]
I agree that the first scenario would be ideal for control and later additions. Putting HTML content blocks into the YAML front matter feels messy and bug prone. It would be nice to be able to add blocks and include them in the template.

The issue I can see arising from this is the need to create a consistent setup for this. Would the entire setup for a block be in the block body? Inheritance of parent properties within the block could cause issues.

Overall, I think its a nice goal.

**Comment by alexsomeoddpilot** [comment]
Content which appears on the listing page for "Work" would likely be put in an `index` file in that project’s subdirectory.

**Comment by krispy1298** [comment]
i think i got a quarter of the way there.

but linking and filtering by project, and detail project page is being tricky.

**Comment by krispy1298** [comment]
i think ive reach my knowledge limit for this issue.

**Comment by alexsomeoddpilot** [comment]
I'm working on grouping collection items in [gulp-collections](https://github.com/alexsomeoddpilot/gulp-collections/tree/grouping). I'll let you know when it has been included.

**Comment by krispy1298** [comment]
i saw your edits for nested pages. right now it doesnt differentiate different client sections. ie. all work sections come up in each client page no matter the client the section is associated with.

**Comment by alexsomeoddpilot** [comment]
I'll work on this.
